client/client.go

```diff
@@ -78,6 +79,18 @@ func (c *Client) prepareRequest(ctx context.Context, req *Request) (*http.Reques
 		return nil, fmt.Errorf("unable to create http request: %w", err)
 	}
 
+	/*
```

Comment:
This code was not ment to be committed. It was part of my test when we had issues with the Content-Length interaction in Go. The measurement service required a Content-Length (but since its not part of the standard we changed it in the service instead of the client)